installer_scripts/install-yb-voyager

```diff
@@ -248,41 +248,54 @@ macos_main() {
+	if [ "${VERSION}" == "latest" ]; then
+		RELEASE_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/releases/latest)
+		if [ -z "$RELEASE_DATA" ]; then
+			echo "ERROR: Failed to fetch the latest release data from the GitHub API."
+			exit 1
+		fi
+	fi 
 
-    # Extract the latest release name and tag name
-    LATEST_RELEASE_NAME=$(echo "$LATEST_RELEASE_DATA" | jq -r '.name')
-    LATEST_TAG_NAME=$(echo "$LATEST_RELEASE_DATA" | jq -r '.tag_name')
+    # Extract the latest release name and tag name from the fetched data if VERSION is latest
+	if [ "${VERSION}" == "latest" ]; then
+		RELEASE_NAME=$(echo "$RELEASE_DATA" | jq -r '.name')
+		TAG_NAME=$(echo "$RELEASE_DATA" | jq -r '.tag_name')
+	else 
+		# If the version is not latest, then the provided VERSION is used in the release name and tag name
+		RELEASE_NAME="v${VERSION}"
+		TAG_NAME="yb-voyager/v${VERSION}"
+	fi
 
-    # Fetch the commit hash of the latest tagged commit
-    LATEST_TAG_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/git/refs/tags/${LATEST_TAG_NAME})
-    if [ -z "$LATEST_TAG_DATA" ]; then
-        echo "ERROR: Failed to fetch the latest tagged commit data from the GitHub API."
+    # Fetch the commit hash of the tagged commit related to the release
+    TAG_DATA=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/git/refs/tags/${TAG_NAME})
+    if [ -z "$TAG_DATA" ]; then
+        echo "ERROR: Failed to fetch the tagged commit data from the GitHub API."
         exit 1
     fi
 
-    LATEST_TAGGED_COMMIT=$(echo "$LATEST_TAG_DATA" | jq -r '.object.sha')
+    TAGGED_COMMIT=$(echo "$TAG_DATA" | jq -r '.object.sha')
 
     # Extract voyager version from the release name
-    VOYAGER_VERSION=$(echo "$LATEST_RELEASE_NAME" | sed 's/v//')
+    VOYAGER_VERSION=$(echo "$RELEASE_NAME" | sed 's/v//')
 
     # Log the fetched data to the log file
-    echo "LATEST_RELEASE_NAME=${LATEST_RELEASE_NAME}" >> "$LOG_FILE"
-    echo "LATEST_TAG_NAME=${LATEST_TAG_NAME}" >> "$LOG_FILE"
-    echo "LATEST_TAGGED_COMMIT=${LATEST_TAGGED_COMMIT}" >> "$LOG_FILE"
+    echo "RELEASE_NAME=${RELEASE_NAME}" >> "$LOG_FILE"
+    echo "TAG_NAME=${TAG_NAME}" >> "$LOG_FILE"
+    echo "TAGGED_COMMIT=${TAGGED_COMMIT}" >> "$LOG_FILE"
     echo "VOYAGER_VERSION=${VOYAGER_VERSION}" >> "$LOG_FILE"
 
     # Set global variables for version and hash
-    VOYAGER_RELEASE_NAME=${VOYAGER_RELEASE_NAME:-${LATEST_RELEASE_NAME}}
-    DEBEZIUM_VERSION=${DEBEZIUM_VERSION:-"2.5.2-${VOYAGER_VERSION}"}
-    YB_VOYAGER_GIT_HASH=${LATEST_TAGGED_COMMIT}
+    VOYAGER_RELEASE_NAME=${RELEASE_NAME}
+    DEBEZIUM_VERSION="${DEBEZIUM_LOCAL_VERSION}-${VOYAGER_VERSION}"
+	# If voyager version contains 0rcx then DEBEZIUM_VERSION will be like 0rcx.2.5.2-[voyager version without rcx]
+	if [[ $VOYAGER_VERSION == *"rc"* ]]; then
+		# In case rc release voyager version is like 0rc1.1.8.5
+		DEBEZIUM_VERSION="${VOYAGER_VERSION:0:4}.${DEBEZIUM_LOCAL_VERSION}-${VOYAGER_VERSION:5}"
```

Comment:
you can probably have this for better understanding -
```
RC_PREFIX=${VOYAGER_RELEASE_VERSION:0:4}
VOYAGER_VERSION=${VOYAGER_RELEASE_VERSION:5}
DEBEZIUM_VERSION="${RC_PREFIX}.${DEBEZIUM_LOCAL_VERSION}-${VOYAGER_VERSION}
```

Reply:
Done